Connnect/JoinViewController.swift

```diff
@@ -16,6 +16,9 @@ class JoinViewController: UIViewController {
     @IBAction func newChannel(sender: UIButton) {
         guard let newChannelName = joinTextField.text else { return }
 
+        guard newChannelName != ""
+            else { return }
+        
```

Comment:
Would be good to inform the user inside the place holder text or somewhere else about the mistake